Source/Prism/Ioc/IContainerProvider.cs

```diff
@@ -10,5 +10,7 @@ public interface IContainerProvider
         object Resolve(Type type, params (Type Type, object Instance)[] parameters);
 
         object Resolve(Type type, string name);
+
+        object Resolve(Type type, string name, params (Type Type, object Instance)[] parameters);
```

Comment:
@totht91 I'd be happy to discuss more with you via Slack... we've already added support for Tuples, you'll notice that this actually just expands on line 10 to resolve named types... ultimately this specific API isn't one that you should be using as it is really more typical for the underlying Prism framework to be resolving types with the container.

Reply:
I would also add, the "don't use tuples in public API" is more about returning a Tuple from a method which is a bad idea because there is no context tot he return values. In this case a tuple is being used as a parameter which is strong typed and named to provide context. We are using Tuples here, so you aren't would be forced to create a dictionary object just to pass in some parameters. This will result in much cleaner code in your code base.